packages/fast-components-react-msft/src/flipper/flipper.tsx

```diff
@@ -34,9 +39,8 @@ class Flipper extends Foundation<FlipperHandledProps, FlipperUnhandledProps, {}>
                 {...this.coerceButtonProps()}
                 className={this.generateClassNames()}
             >
-                <span className={get(this.props, "managedClasses.flipper_glyph")}>
-                    {this.props.children}
-                </span>
+                {this.renderGlyph()}
+                {this.props.children}
```

Comment:
wait, did we want to render children in here? I remember this being a bug, which we may was well fix here.

Reply:
The bug was that it (say text) was rotated with the fake glyph. The styling isn't correct as-is, but rendering children would allow someone to override the styling as necessary. It's probably not common, but I think it should still be supported.